### ISSUE TITLE
Lighting throughout the house

### ISSUE THREAD

**Issue by iteles** [dwyl/home#47]
We need to finalise and buy light fixtures for:
  + [x] Work areas
  + [x] Kitchen
  + [x] Sleeping areas
  + [x] Communal/passage areas
  + [x] Bathrooms - moved to #72 
+ [x] Final lights from https://github.com/dwyl/home/issues/47#issuecomment-564088396 installed

@nelsonic We've spoken about this before and you've already done a lot of research into it already. Want to drop it into here? 😊

**Comment by iteles** [comment]
I really like the feel of having some statement lighting like this:
![image](https://user-images.githubusercontent.com/4185328/54087040-1379d000-4347-11e9-8dcc-ee8109e3ce77.png)

[![image](https://user-images.githubusercontent.com/4185328/54087050-28566380-4347-11e9-8f27-795c0d0ecca8.png)](http://www.leroymerlin.pt/Site/Produtos/Iluminacao/Candeeiros-suspensos/Suspensoes/81926848.aspx)

But it doesn't make sense in the vast majority of the house.   
The only places I could see it working would be the library and the dining room.

They _could_ make great corridor lights (and would look really cool from the outside looking in) but I wonder if they'd give _enough_ light?

**Comment by iteles** [comment]
We went with fully adjustable lighting in the _entire_ house (to promote better sleep cycles) except in two areas, where we went with the more statement lights above:
+ Dining room 
+ Entryway corridor 

🎉

**Comment by iteles** [comment]
There are two places we still need to buy lights for:
+ The stairway down to the basement
+ The stairway up to the first floor
+ The bathrooms I think!
But I'm very much struggling to find lights that will look decent!

## Stairs to basement
The wall space here is very small so we should look to keep them as flush to the wall as possible, but still providing decent lighting


## Stairs from the entrance
These stairs will be book-ended by statement lights at the bottom (in the entrance hallway) and long rectangular LED panels in the top corridor (which we might want to reconsider for consistency...)

They therefore need to be:
+ Super simple
+ Provide ample light and therefore cannot be a 'foco'
+ Be either black or completely white

But I haven't really found anything pre-existing that I like 😕 Here are some examples:
[![image](https://user-images.githubusercontent.com/4185328/58369463-88928580-7ef2-11e9-8eca-541e5928f801.png)](https://www.graceandgloryhome.co.uk/molecular-wall-lamp.html)

[![image](https://user-images.githubusercontent.com/4185328/58369479-d3ac9880-7ef2-11e9-8303-d44b38738139.png)](http://www.leroymerlin.pt/Site/Produtos/Iluminacao/Focos/Focos-de-superficie/19331004.aspx)

[![image](https://user-images.githubusercontent.com/4185328/58369595-892c1b80-7ef4-11e9-943e-8de414c2aac5.png)](https://www.mundiluz.pt/apliques-parede/836-aplique-parede-philips-gainsboro-1x23w-8718291455714.html)

This one can be regulated with a (sold separately) remote control:
[![image](https://user-images.githubusercontent.com/4185328/58369560-158a0e80-7ef4-11e9-8d29-22a18ead536b.png)](https://www.mundiluz.pt/apliques-parede/872-aplique-parede-philips-homroo-1x60w-8718291463351.html)



![image](https://user-images.githubusercontent.com/4185328/58369664-57678480-7ef5-11e9-8316-063bff7c6556.png)

**Comment by iteles** [comment]
We went to our trusty Leroy Merlin and got the these for the main stairwell to match our corridor lighting (which I hadn't found online before but spotted in the shop):

[![image](https://user-images.githubusercontent.com/4185328/58762532-ac844580-8548-11e9-9e13-5edb1488091e.png)](http://www.leroymerlin.pt/Site/Produtos/Iluminacao/Apliques-e-plafons/81944394.aspx)

<hr/>

And these for the basement stairwell because we needed lights to protrude as little as possible. 
We'll test them to make sure they have enough light.

![IMG_0858](https://user-images.githubusercontent.com/4185328/58762431-e4d75400-8547-11e9-9f20-97b97f6928af.JPG)

<hr/>

We pretty much have this issue covered now except bathrooms and exterior lighting #61 .

I'm opening a separate issue on bathroom lighting and believe we can close this one to avoid endless scrolling.

**Comment by iteles** [comment]
@nelsonic We are likely to get a half day from the electricians before the end of the week to finish adding the electric wiring to the doors.

They won't be able to do _much_ in terms of putting light fittings in, but we should decide which rooms we'd like them to prioritise, i.e. which ones we'll use. I'm thinking an upstairs bedroom and then a couple of rooms in the basement that are to be used for machinery (for now the garage and the MR) - these could have light fittings and sockets installed throughout for ease of use in August.

The electricians then go on holiday from the end of this week until the end of August.

**Comment by nelsonic** [comment]
@iteles I don't understand _why_ you have assigned this issue to me.
If you've already decided which rooms they should focus on adding lights to, then just tell them.

**Comment by iteles** [comment]
Let me be clear: You are the person who will determine which rooms are required for the machinery in the immediate term. I have _suggested_ the garage and reduced mobility room but you may prefer the media centre for example.

Please confirm which rooms you would choose to prioritise in the immediate term.

**Comment by nelsonic** [comment]
@iteles as long as I have access to electricity (_which I do_) I can use my photography studio lights to illuminate anything I'm working on in the workshop (_as I have always done in London..._).
Furthermore I have a portable generator which means I can work 100% off-grid.
Lighting and electrical sockets are 100% not a "bottleneck" for me. 

If _you_ need/want specific lights to be available in the house, please inform them.

**Comment by nelsonic** [comment]
# Clarification of Lighting _Throughout_ the House

Our plan has _always_ been to use **sensors** for **all _communal_ areas** in the house 
and write _bespoke_ software to control each light throughout the day 
to ensure they are in harmony with the circadian rhythm.
The software is _incredibly_ simple and I can write in one (_focussed_) day (_including tests and docs!).
It literally is a single switch statement that checks the time of day and applies the appropriate signal to a relay; a 6 year old could write it in Scratch in half an hour! The only reason I haven't _already_ done it is that we have _many_ other/higher priorities and custom lighting algorithms are _not_ a dependency for us moving into the house. The most "complex" part is creating an interface to control the lights, but that is 100% optional as I'm not convinced we _need_ the ability to administer the lights _centrally_ if they all operate on motion detectors on location. If having an interface becomes a priority (_for whatever reason people want finer control_)  I'm leaning towards using Home Assistant which is 100% open source and local-network bound.
https://www.home-assistant.io + https://github.com/home-assistant/home-assistant
![image](https://user-images.githubusercontent.com/194400/65340838-0035a000-dbc7-11e9-9a16-bfde6ff22060.png)

So, to summarise: the software can be written in a day and applied to _all_ lights in the house.
The `default` will be 100% brightness at all times until will refine the code over time.

**Comment by iteles** [comment]
Booked in lighting installation with electricians for next Tuesday (15-Oct-2019), their earliest availability.

**Comment by iteles** [comment]
Delayed by one day to Wednesday 16-Oct-2019.

**Comment by iteles** [comment]
Final list of things needed:
- 3 super SHORT length sensors (boiler room, storage above 'house masters' room that we don’t want to go off every time they move in bed, storage room in attic)
- 2 normal sensors (attic main light and basement steps)
- 2 round lights that sits ONTOP of the ceiling (boiler room and attic storage room)
- 2 round light NON temperature controlled  (attic ‘corridor’)
- 1 round light adjustable (attic storage above 'house masters' room)

These were all bought and dropped off at the house on 5-Dec-2019.

All other lights now needed are external 👍

**Comment by iteles** [comment]
These were all installed today, 14-Jan-2020, with a couple of changes:
+ The storage above the housemasters' room and the storage in the attic both have switches and not sensors
+ The attic 'corridor' requires 2 sensors instead of only one (one for each light) and these have been installed today as well.

**Comment by iteles** [comment]
These are now all done, we will go through the house and open a separate snag list issue if needed.
